tensorflow/contrib/cmake/tf_python.cmake

```diff
@@ -957,7 +960,7 @@ add_library(pywrap_tensorflow_internal SHARED
     $<TARGET_OBJECTS:tf_tools_transform_graph_lib>
     $<$<BOOL:${tensorflow_ENABLE_GRPC_SUPPORT}>:$<TARGET_OBJECTS:tf_core_distributed_runtime>>
     $<TARGET_OBJECTS:tf_core_kernels>
-    $<$<BOOL:${tensorflow_ENABLE_GPU}>:$<TARGET_OBJECTS:tf_core_kernels_cpu_only>>
+    $<$<BOOL:${tensorflow_ENABLE_GPU}>:$<$<BOOL:${BOOL_WIN32}>:$<TARGET_OBJECTS:tf_core_kernels_cpu_only>>>
```

Comment:
I'm unsure why this particular target is WIN32 only. Can you please add a comment indicating why this is platform-dependent?

Reply:
That ```tf_core_kernels_cpu_only``` is for Windows only, which has sort of "workaround" for Windows that does not support GPU in specific features. In Linux, this ```tf_core_kernel_cpu_only``` is not necessary.